mythx_cli/analyze/command.py

```diff
@@ -170,87 +204,45 @@ def analyze(
 
     jobs: List[Dict[str, Any]] = []
     include = list(include)
+    mode_list = determine_analysis_targets(target, forced_scenario=scenario)
 
-    if not target:
-        if Path("truffle-config.js").exists() or Path("truffle.js").exists():
-            files, source_list = find_truffle_artifacts(Path.cwd())
-            if not files:
-                raise click.exceptions.UsageError(
-                    (
-                        "Could not find any truffle artifacts. Are you in the project root? "
-                        "Did you run truffle compile?"
-                    )
+    for analyze_mode, element in mode_list:
```

Comment:
This is just nitpicking and probably not actionable, but `analyze_mode` might be confused with the analyses mode. Maybe something like `project_mode` would be better 🤔

Reply:
Good catch! This is an old name from early development. I have renamed the tuple element `scenario` and the related enum holding the types `ScenarioModes` in 2530b53